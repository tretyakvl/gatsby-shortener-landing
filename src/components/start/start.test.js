import React from 'react'
import renderer from 'react-test-renderer'
import Start from './start'

describe('Start', () => {
  it('renders correctly', () => {
    const props = {
      title: 'Start spending with us!',
      cta: {
        to: '/buy-the-product-please',
        text: 'No refunds, seriously'
      }
    }

    const tree = renderer
      .create(<Start {...props} />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
