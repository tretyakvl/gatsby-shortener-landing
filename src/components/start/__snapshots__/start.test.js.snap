// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Start renders correctly 1`] = `
<section
  className="start"
>
  <div
    className="start__container"
    style={
      Object {
        "opacity": 0,
        "transform": "translateX(-20px) translateZ(0)",
      }
    }
  >
    <h2>
      Start spending with us!
    </h2>
    <a
      className="button-module button"
      href="/buy-the-product-please"
    >
      No refunds, seriously
    </a>
  </div>
</section>
`;
